lab2/src/main/java/table/SymbolTable.java

```diff
@@ -11,6 +11,8 @@
     private List<List<String>> elements;
 
 
+
+
```

Comment:
Seems fine, maybe rename the hashCode function because it sounds more like a variable than a function.